src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -287,6 +289,10 @@ public enum CassandraRelevantProperties
     IO_NETTY_EVENTLOOP_THREADS("io.netty.eventLoopThreads"),
     IO_NETTY_TRANSPORT_ESTIMATE_SIZE_ON_SUBMIT("io.netty.transport.estimateSizeOnSubmit"),
     IO_NETTY_TRANSPORT_NONATIVE("io.netty.transport.noNative"),
+    JAVAX_NET_SSL_KEYSTORE("javax.net.ssl.keyStore"),
```

Comment:
This is great @smiklosovic! So now does it make sense to update `JMXSslConfigDistributedTest#setKeystoreProperties()` method to use this also? instead of hardcoded strings there?

Reply:
right, fixed.